examples/misc/keyword_plotting.py

```diff
@@ -0,0 +1,22 @@
+"""
+Plotting with keywords
+======================
+
+There are some instances where you have data in a format that lets you
+access particular variables with strings. For example, with a ``recarray``
```

Comment:
At the risk of doing the rst markup police again and again: optimally, `recarray` and `DataFrame` should be in single backquotes.  See e.g. https://raw.githubusercontent.com/anntzer/mplcursors/master/doc/index.rst, look for
```
:mod:`pandas`’ :class:`DataFrame <pandas.DataFrame>`\s,
```
rendered at http://mplcursors.readthedocs.io/en/latest/.

(Also check https://github.com/anntzer/mplcursors/blob/master/doc/conf.py#L349).

See also https://mail.zope.org/pipermail/grok-dev/2008-December/006823.html, https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#inline-markup-and-special-characters-e-g-bold-italic-verbatim.